### PR TITLE
Remove unused CASTOR tags and change Castor DQM to Run-3 era

### LINKED ISSUE #22: Remove Castor tags
* CASTOR is decommissioned, can be removed from Run3 specific tags
 * Should be understood if there are code changes required
```
CastorChannelQualityRcd
CastorElectronicsMapRcd
CastorGainsRcdCastorGainWidthsRcd
CastorPedestalsRcd
CastorPedestalWidthsRcd
CastorQIEDataRcd
CastorRecoParamsRcd
CastorSaturationCorrsRcd
```

### PR DESCRIPTION
**Pull request by tvami**
#### PR description:

Remove unused CASTOR tags and change Castor DQM to Run-3 era

New GTs and their diffs
* 123X_dataRun3_HLT_v5
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/123X_dataRun3_HLT_v4/123X_dataRun3_HLT_v5
 * 123X_dataRun3_Express_v4
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/123X_dataRun3_Express_v3/123X_dataRun3_Express_v4
* 123X_dataRun3_Prompt_v4
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/123X_dataRun3_Prompt_v3/123X_dataRun3_Prompt_v4
 * 123X_dataRun3_v3
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/123X_dataRun3_v2/123X_dataRun3_v3
 * 123X_mcRun3_2021_design_v8
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/123X_mcRun3_2021_design_v7/123X_mcRun3_2021_design_v8
 * 123X_mcRun3_2021_realistic_v8
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/123X_mcRun3_2021_realistic_v7/123X_mcRun3_2021_realistic_v8
 * 123X_mcRun3_2021cosmics_realistic_deco_v8
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/123X_mcRun3_2021cosmics_realistic_deco_v7/123X_mcRun3_2021cosmics_realistic_deco_v8
 * 123X_mcRun3_2021_realistic_HI_v8
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/123X_mcRun3_2021_realistic_HI_v7/123X_mcRun3_2021_realistic_HI_v8
 * 123X_mcRun3_2023_realistic_v8
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/123X_mcRun3_2023_realistic_v7/123X_mcRun3_2023_realistic_v8
 * 123X_mcRun3_2024_realistic_v8
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/123X_mcRun3_2024_realistic_v7/123X_mcRun3_2024_realistic_v8
 * 123X_mcRun4_realistic_v6
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/123X_mcRun4_realistic_v5/123X_mcRun4_realistic_v6

Diffs are in the castor tags + the HI queue had a tag by mistake that's removed here (should have no effect)

#### PR validation:

136.793,136.874,140.56,12034.0,7.23,159.0,12834.0,136.8642,136.7952

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

N/A

resolves https://github.com/cms-AlCaDB/AlCaTools/issues/22

### CODE REVIEW

**Comment by tvami** [comment]
test parameters:
  - workflows = 136.793,136.874,140.56,12034.0,7.23,159.0,12834.0,136.8642,136.7952

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by tvami** [comment]
This will conflict with https://github.com/cms-sw/cmssw/pull/37034 , but that's currently hold back by the reco signature so I'm launching this as a first check, and then will rebase after the other one has merged....

**Comment by tvami** [comment]
@cmsbuild , please test

File: DQM/Integration/python/clients/castor_dqm_sourceclient-live_cfg.py [hunk 1]
```diff
@@ -2,7 +2,8 @@
 import FWCore.ParameterSet.Config as cms
 import sys
 
-process = cms.Process("CASTORDQM")
+from Configuration.Eras.Era_Run3_cff import Run3
+process = cms.Process("CASTORDQM", Run3)
```
Comment (mmusich) [inline DQM/Integration/python/clients/castor_dqm_sourceclient-live_cfg.py:6]: out of curiosity, what's the point of this client if CASTOR doesn't exist in Run3?

Comment (tvami) [reply DQM/Integration/python/clients/castor_dqm_sourceclient-live_cfg.py:6]: nothing... I'll remove it instead of trying to fix it with eras

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by tvami** [comment]
So I do have to go back to a previous state when these configs had the Run3 era. I'll do that tomorrow

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by tvami** [comment]
+alca
 * no diffs are seen, as expected

**Comment by jfernan2** [comment]
@tvami when you say "no diffs are seen", are you aware that 37 MEs are being removed in this PR for CASTOR folder?
https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_12_3_X_2022-02-26-1100+96ce97/48592/dqm-histo-comparison-summary.html
Though it is true that most of them were empty.... Thanks

**Comment by tvami** [comment]
Thanks @jfernan2 yes, this is fine as there is no Castor anymore, so we for sure dont need any DQM plots from Run-3 on about it.

**Comment by jfernan2** [comment]
+1

File: Configuration/AlCa/python/autoCond.py [hunk 2]
```diff
@@ -31,16 +31,16 @@
     'run2_data_relval'             : '123X_dataRun2_relval_v2',
     # GlobalTag for Run2 HI data
     'run2_data_promptlike_hi'      : '123X_dataRun2_PromptLike_HI_v2',
-    # GlobalTag for Run3 HLT: it points to the online GT
-    'run3_hlt'                     : '123X_dataRun3_HLT_v4',
     # GlobalTag with fixed snapshot time for Run2 HLT RelVals: customizations to run with fixed L1 Menu
-    'run2_hlt_relval'              : '123X_dataRun2_HLT_relval_v2',
+    'run2_hlt_relval'              : '123X_dataRun2_HLT_relval_v1',
```
Comment (perrotta) [inline Configuration/AlCa/python/autoCond.py:35]: Is this going back in the version number wanted?

Comment (tvami) [reply Configuration/AlCa/python/autoCond.py:35]: No, that should have stayed the v2, I'll fix it

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by jfernan2** [comment]
+1

**Comment by francescobrivio** [comment]
+alca

**Comment by perrotta** [comment]
+1

**Pull request merged**